app/page.tsx

```
@@ -6,13 +6,15 @@ import Developers from "@/components/home/developers";
 import HowItWorks from "@/components/home/how-it-works";
 import FAQ from "@/components/home/faq";
 import Footer from "@/components/footer";
+import { useIsAuthenticated } from "./auth/hooks/use-is-authenticated";
 
-export default function Home() {
+export default async function Home() {
+  const isAuthenticated = await useIsAuthenticated();
```

ESLint

[error] React Hook "useIsAuthenticated" cannot be called in an async function.

(react-hooks/rules-of-hooks)